package.json

```diff
@@ -9,48 +9,47 @@
     "url": "https://liqd.net"
   },
   "dependencies": {
-    "@babel/core": "7.12.13",
-    "@babel/plugin-transform-modules-commonjs": "7.12.13",
-    "@babel/plugin-transform-runtime": "7.12.15",
-    "@babel/preset-env": "7.12.13",
-    "@babel/preset-react": "7.12.13",
-    "@babel/runtime": "7.12.13",
-    "@fortawesome/fontawesome-free": "5.15.2",
+    "@babel/core": "7.17.8",
+    "@babel/plugin-transform-modules-commonjs": "7.17.7",
+    "@babel/plugin-transform-runtime": "7.17.0",
+    "@babel/preset-env": "7.16.11",
+    "@babel/preset-react": "7.16.7",
+    "@babel/runtime": "7.17.8",
+    "@fortawesome/fontawesome-free": "5.15.4",
     "adhocracy4": "github:liqd/adhocracy4#7a6ad7685f489ed7c06c19bbf456760755b4eca7",
-    "autoprefixer": "10.2.4",
-    "axios": "0.21.1",
-    "babel-loader": "8.2.2",
-    "bootstrap": "4.6.0",
-    "classnames": "2.2.6",
-    "copy-webpack-plugin": "5.1.2",
-    "css-loader": "5.0.1",
-    "extract-text-webpack-plugin": "3.0.2",
+    "autoprefixer": "10.4.4",
+    "babel-loader": "8.2.4",
+    "bootstrap": "4.6.1",
+    "classnames": "2.3.1",
+    "copy-webpack-plugin": "10.2.4",
+    "css-loader": "6.7.1",
+    "extract-text-webpack-plugin": "^3.0.2",
```

Comment:
I think we can remove that. It's been deprecated since 2017 and was replaced by ``mini-css-extract-plugin`` already

Reply:
oh, actually building fails without it, ouch.